internal/controller/system/module.go

```diff
@@ -34,21 +34,27 @@ func NewFXModule(configuration ModuleConfiguration) fx.Option {
 			meterProvider metric.MeterProvider,
 			tracerProvider trace.TracerProvider,
 		) *DefaultController {
-			var parser ledgercontroller.NumscriptParser = ledgercontroller.NewDefaultNumscriptParser()
-			if configuration.NumscriptInterpreter {
-				parser = ledgercontroller.NewInterpreterNumscriptParser(configuration.NumscriptInterpreterFlags)
-			}
+			var machineParser ledgercontroller.NumscriptParser = ledgercontroller.NewDefaultNumscriptParser()
```

Comment:
Not a big deal, but I try to group variables declaration using blocks. Like that : 
```
var (
    a string
    b string
)
```